pkg/cmd/cli/cmd/newapp.go

```diff
@@ -99,6 +101,16 @@ func NewCmdNewApplication(fullName string, f *clientcmd.Factory, out io.Writer)
 				glog.Fatalf("Error: %v", err)
 			}
 
+			label := cmdutil.GetFlagString(c, "label")
+			if len(label) == 0 {
+				label = util.GetLabel()
```

Comment:
Selector is for finding things by name.  Labels are what you are adding.  It should be "labels"


> On Apr 2, 2015, at 4:21 AM, Michail Kargakis <notifications@github.com> wrote:
> 
> In pkg/cmd/cli/cmd/newapp.go:
> 
> > @@ -99,6 +101,16 @@ func NewCmdNewApplication(fullName string, f *clientcmd.Factory, out io.Writer)
> >  				glog.Fatalf("Error: %v", err)
> >  			}
> >  
> > +			label := cmdutil.GetFlagString(c, "label")
> > +			if len(label) == 0 {
> > +				label = util.GetLabel()
> Ah, wording. The flag should be "--labels"
> 
> I am going to change it to "--selector" as kubectl delete uses it.
> 
> —
> Reply to this email directly or view it on GitHub.
>